tests/test_import_spark.py

```diff
@@ -0,0 +1,162 @@
+import yaml
+import pytest
+from pyspark.sql import types
+
+from pyspark.sql import SparkSession
+
+from datacontract.data_contract import DataContract
+
+from typer.testing import CliRunner
+from datacontract.cli import app
+
+
+expected = """
+dataContractSpecification: 0.9.3
+id: my-data-contract-id
+info:
+  title: My Data Contract
+  version: 0.0.1
+servers:
+  local:
+    type: dataframe
```

Comment:
I think we should not add this here, as there is no server type `dataframe` at the moment at datacontract.com

Reply:
Yes, there is. At least in the schema.